### DISCUSSION TITLE
Wie komme ich zur richtigen Vorhersage

### DISCUSSION THREAD

**Discussion by ManfredK1968** [roe-dl/weewx-DWD#72] [General]
Hallo,
Ich probiere jetz schon einige Zeit mit der DWD-Erweiterung herum, und wollte die Vorhersage für Tulln (Niederösterreich) erstellen.
Ich habe jetzt folgende Anpassungen durchgeführt:

weewx.conf:
#############################################################################

[WeatherServices]
    # Verzeichnis, in das die Dateien gespeichert werden sollen
    path='/home/pi/weewx-data/skins/Belchertown/dwd'
    [[download]]
        [[[Download1]]]
            # Bodenwetterkarte vom DWD
            provider = DWD
            model = bwk-map
    [[Belchertown]]
        section = "BelchertownReport"
        warnings = DL
        forecast = 11030
    [[forecast]]
        # Speicherort der Symbole auf dem Web-Server
        icons = '../images'
        # zu verwendender Symbolsatz: belchertown, dwd, aeris
        icon_set = dwd
        # für welche Orientierungen soll die HTML-Datei erzeugt werden?
        #orientation = h,v
        # Sollen die Meßgrößensymbole in die Tabelle aufgenommen werden?
        #show_obs_symbols = True # optional
        # Soll die Meßgrößenbeschreibung in die Tabelle aufgenommen werden?
        #show_obs_description = False # optional
        # Soll der Ortsname über die Tabelle geschrieben werden?
        #show_placemark = True # optional
        # Beispiele
        [[[11030]]]
            # Wettervorhersage
            provider = DWD
            model = MOSMIX
            # optional darzustellende Wetterelemente anstelle des defaults
            #observations_daily = ww, TTTmax, TTTmin, windchill, FFavg, DDavg, RR1c, Rd10, Neffavg, RSunD, Rad1hsum, VVmin
            #observations_hourly = TTT, humidity , FF , DD , RR1c , R101 , PPPP , Rad1h
            # Was würde eine PV-Anlage an dem Tag bringen? 
            #pv_factor = 10.0 # Fläche * Wirkungsgrad            
#############################################################################

wenn ich nun im Terminal folgende Aufruf starte, erhalte ich jedoch immer die 'forecast-P0291.inc' (Vorhersage für Döbeln) im dwd-Skin Ordner.

> dwd-mosmix --config=/home/pi/weewx-data/weewx.conf --html

Was habe ich übersehen, bzw. was mache ich falsch?
Dank im Voraus.

LG
Manfred

**Comment by roe-dl** [comment]
Da fehlt die `11030` am Ende des Aufrufs, also so:

```shell
dwd-mosmix --config=/home/pi/weewx-data/weewx.conf --html 11030
```

**Reply by ManfredK1968** [reply]
Vielen Dank für den Hinweis ;).
